capella_ros_tools/__main__.py

```diff
@@ -149,21 +167,67 @@ def import_msgs(
     default=pathlib.Path.cwd() / "data-package",
     help="Output directory for the .msg files.",
 )
-def export_capella(
+@click.option(
+    "--generate-cmake",
+    type=bool,
+    default=False,
+    is_flag=True,
+    help="Decide whether experimental cmake files should be generated.",
+    envvar="CAPELLA_ROS_TOOLS_GENERATE_CMAKE",
+)
```

Comment:
I'm a fan of short flag names. And short help texts. :)


```suggestion
@click.option(
    "--cmake",
    type=bool,
    default=False,
    is_flag=True,
    help="Generate cmake files. Experimental.",
    envvar="CAPELLA_ROS_TOOLS_GENERATE_CMAKE",
)
```